style/components/button.css

```diff
@@ -0,0 +1,100 @@
+.btn {
+  padding: 0.8rem 1rem;
+  border: none;
+  cursor: pointer;
+  font-size: 18px;
+  color: white;
+  background-color: gray;
+  margin-left: 0.5rem;
+  margin-right: 0.5rem;
+  transition: 0.2s;
+}
+
+.primary {
```

Comment:
You can add the prefix `btn` or` button` , `btn-primary` is clearer. You can use this convention for all the classes if you want to.

Reply:
sure doing the required change